auth0_flutter/lib/src/mobile/authentication_api.dart

```diff
@@ -149,6 +149,95 @@ class AuthenticationApi {
+  ///
+  /// Your Application must have the **Passwordless OTP** Grant Type enabled.
+  ///
+  /// ## Usage example:
+  /// ```dart
+  /// final result = await auth0.api.startPasswordlessWithEmail({
+  ///   email : 'email',
+  ///   passwordlessType : 'PasswordlessType'
+  ///   });
+  /// ```
+  ///
+  Future<void> startPasswordlessWithEmail(
+          {required final String email,
+          required final PasswordlessType passwordlessType}) =>
+      Auth0FlutterAuthPlatform.instance.passwordlessWithEmail(_createApiRequest(
+          AuthPasswordlessLoginOptions(
+              email: email, passwordlessType: passwordlessType)));
+
+  /// Log in a user using an email and a verification code received via Email (Part of passwordless login flow).
+  /// The default scope used is 'openid profile email'.
+  ///
+  /// Your Application must have the **Passwordless OTP** Grant Type enabled.
+  ///
+  /// ## Usage example:
+  /// ```dart
+  /// final result = await auth0.api.loginWithEmail({
+  ///   email: 'email',
+  ///   verificationCode: 'code'
+  /// });
+  ///```
+  Future<Credentials> loginWithEmail(
+          {required final String email,
+          required final String verificationCode,
+          final String? connection,
+          final String? scope,
+          final String? audience}) =>
+      Auth0FlutterAuthPlatform.instance.loginWithEmail(_createApiRequest(
+          AuthPasswordlessLoginOptions(
+              email: email,
+              verificationCode: verificationCode,
+              connection: connection,
+              scope: scope,
+              audience: audience)));
+
+  /// Start a passwordless flow with a [SMS](https://auth0.com/docs/api/authentication#get-code-or-link)
+  ///
+  /// Your Application requires to have the **Passwordless OTP** Grant Type enabled.
+  ///
+  /// ## Usage example:
+  /// ```dart
+  /// final result = await auth0.api.startPasswordlessWithPhoneNumber({
+  ///   phoneNumber : 'phoneNumber',
+  ///   passwordlessType : 'PasswordlessType'
+  ///   });
+  /// ```
+  Future<void> startPasswordlessWithPhoneNumber(
+          {required final String phoneNumber,
+            required final PasswordlessType passwordlessType}) =>
+      Auth0FlutterAuthPlatform.instance.passwordlessWithPhoneNumber(
+          _createApiRequest(AuthPasswordlessLoginOptions(
+              phoneNumber: phoneNumber,passwordlessType: passwordlessType)));
+
+  /// Log in a user using a phone number and a verification code received via SMS (Part of passwordless login flow).
+  /// The default scope used is 'openid profile email'.
+  ///
+  /// Your Application must have the **Passwordless OTP** Grant Type enabled.
+  ///
+  /// ## Usage example:
+  /// ```dart
+  /// final result = await auth0.api.loginWithPhoneNumber({
+  ///   phoneNumber: 'phoneNumber',
+  ///   verificationCode: 'code'
+  /// });
+  ///```
+  Future<Credentials> loginWithPhoneNumber(
```

Comment:
As mentioned in [startPasswordlessWithPhoneNumber](https://github.com/auth0/auth0-flutter/pull/503/files#r1935356404), this one should probably be called `loginWithSmsCode`